Library/Formula/h2o.rb

```diff
@@ -0,0 +1,74 @@
+class H2o < Formula
+  homepage "https://github.com/h2o/h2o/"
+  url "https://github.com/h2o/h2o/archive/v0.9.0.tar.gz"
+  sha1 "37b84750900cdb56c3be574477d6f3327d92a4d7"
+  head "https://github.com/h2o/h2o.git"
+
+  option "with-libuv", "Build the H2O library as well as the executable."
+
+  depends_on "cmake" => :build
+  depends_on "libyaml"
+  depends_on "openssl"
+  depends_on "libuv" => :optional
+
+  def install
+    system "cmake", ".", *std_cmake_args
+
+    if build.with? "libuv"
+      system "make", "libh2o"
+      lib.install "libh2o.a"
+    end
+
+    system "make", "install"
+
+    mkdir_p etc/"h2o"
+    mkdir_p var/"h2o"
+    (var+"h2o").install "examples/doc_root/index.html"
+    (etc+"h2o/h2o.conf").write conf_example
+  end
+
+  def conf_example; <<-EOS.undent
```

Comment:
A comment above here saying "this is based on `examples/blah/blah.conf` or similar would be great.